cornflow-dags/DAG/auto_scripts/automatic_scripts.py

```diff
@@ -0,0 +1,166 @@
+import os
+import shutil
+
+from sqlalchemy import create_engine
+from sqlalchemy.exc import SQLAlchemyError
+from datetime import datetime, timedelta
+from airflow import DAG
+from airflow.operators.python import PythonOperator
+
+# Default arguments of DAG
+default_args = {
+    "owner": "baobab",
+    "depends_on_past": False,
+    "start_date": datetime(2023, 1, 1),
+    "email": [""],
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": -1,
+    "retry_delay": timedelta(minutes=1),
+    "catchup": False,
+}
+# Folder path
+SCRIPT_FOLDER = "/usr/local/airflow/dags/auto_scripts/scripts_to_execute"
+DESTINATION_FOLDER = "/usr/local/airflow/dags/auto_scripts/executed_scripts"
+CONN = os.environ.get("CORNFLOW_DB_URI", "NA")
+
+
+def move_script(auto_script, log="", error=0):
+    # Get current date and time
+    now = datetime.now()
+
+    # Format date and time as string
+    datetime_str = now.strftime("%Y%m%d_%H%M")
+
+    # Name of folder to create. If the script had an error, the name of the folder flags it
+    if error:
+        folder_name = f"ERROR_{os.path.splitext(auto_script)[0]}_{datetime_str}"
+    else:
+        folder_name = f"{os.path.splitext(auto_script)[0]}_{datetime_str}"
+
+    # Full path of folder to create
+    folder_path = os.path.join(DESTINATION_FOLDER, folder_name)
+
+    # Create folder if it doesn't exist
+    if not os.path.exists(folder_path):
+        os.mkdir(folder_path)
+        print(f"Folder created successfully at {folder_path}")
+    else:
+        print(f"Folder '{folder_path}' already exists.")
+
+    # Move script to folder
+    script_path = os.path.join(SCRIPT_FOLDER, auto_script)
+    shutil.move(script_path, folder_path)
+    # Export log
+    log_path = os.path.join(folder_path, "log.txt")
+    with open(log_path, "w") as file:
+        # Write log
+        file.write(log)
+    print(f"Script {auto_script} moved out of folder")
+
+
+def execute_scripts():
+    # Iterate directory
+    for auto_script in os.listdir(SCRIPT_FOLDER):
+        error = 0
+        print(f"Executing {auto_script}")
+        # Check if current path is a file
+        if os.path.isfile(os.path.join(SCRIPT_FOLDER, auto_script)):
+            script_path = os.path.join(SCRIPT_FOLDER, auto_script)
+
+            # Check if python file
+            if ".py" in auto_script:
+                error = os.system(f"python {script_path} > output.txt 2>&1")
+
+                with open("output.txt", "r") as f:
+                    log = f.read()
+
+                os.remove("output.txt")
+
+                if error != 0:
+                    print(f"Something went wrong: {str(log)}")
+
+                error = 1
```

Comment:
I think you forgot to remove that last line, `error = 1`

Reply:
@JaimeSotomayor fix this